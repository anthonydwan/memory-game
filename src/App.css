html,
body {
  background-color: rgb(31, 31, 31);
  color: #ebddbb;
}

.App {
  text-align: center;
  max-width: 900px;
  margin: 10vh auto;
  padding: 80px;
  border: 20px solid #db1f48;
  border-radius: 5em;
  background-color: #004369;
  position: relative;
}
.github {
  position: absolute;
  width: 35px;
  right: 60px;
  bottom: 30px;
  transition: all 0.5s ease-in-out;
}

.github:hover,
.github:focus,
.github:active {
  transform: rotate(360deg);
}

.scoreContainer {
  display: flex;
  width: 80%;
  justify-content: space-around;
  margin: auto;
}

.cards {
  width: 20%;
  height: auto;
}

.buttonSection {
  height: 50px;
  margin-top: 20px;
  width: auto;
  display: block;
}

h1 {
  font-size: 56px;
}

span {
  width: 250px;
  font-size: 30px;
  border: none;
  padding: 10px;
  border-radius: 10px;
  background-color: #03293f;
}

button {
  transition: all 0.25s ease-in;
  border: none;
  height: 50px;
  width: 250px;
  font-size: 30px;
  border-radius: 10px;
  color: rgb(31, 31, 31);
  background-color: rosybrown;
}
