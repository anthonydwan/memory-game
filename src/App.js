import "./App.css";
import MainComp from "./components/MainComp";


function App(props) {
  return (
    <div className="App">
      <MainComp/>
    </div>
  );
}

export default App;
