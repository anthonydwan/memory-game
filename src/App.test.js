import { render, screen } from "@testing-library/react";
import App from "./App";

test("renders", () => {
  render(<App />);
  expect(asFragment()).toMatchSnapshot();
});

