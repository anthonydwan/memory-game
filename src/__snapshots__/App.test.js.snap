// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <h1>
      Hi
    </h1>
  </div>
</DocumentFragment>
`;
